component-classes/scripts/updateClassesToPurge.js

```diff
@@ -0,0 +1,11 @@
+import fs from 'node:fs';
+import { classes } from '../classes.js';
+
+export default () => {
+  fs.writeFileSync('./classesToPurge.json', JSON.stringify(classes), 'utf-8');
```

Comment:
Do we have use for this file? Can't find any mentions of `./component-classes/classesToPurge.json` in this repo.

Reply:
We publish this file to Eik/gh pages

Reply:
I thought we are publishing the one that is saved in the next line, under `../dist/classesToPurge.json`.

Reply:
Also, do we still want to publish this to GH pages?

Reply:
I will remove publishing after this PR is merged. No we don't need it anymore

Reply:
Refactored here. I thought that we can commit the file but yea, maybe it is needed. https://github.com/warp-ds/css/pull/56/commits/ec090b2ae41b4340c806f380800c763fad5932b1